### ISSUE TITLE
Started work on NLP Stuff.

### ISSUE THREAD

**Issue by elaksana** [kwhitehall/scored#28]
1. Performed first two NLP tasks with Stanford's coreNLP library.
2. Top N wordcount with stemming, same caps, etc have been completed.
3. Finding datasets (with a not so good heuristic) works.  
4. Some issues with coreNLP timing out on the works cited page.

**Comment by chrismattmann** [comment]
how are you doing NLP? Are you using [Tika CoreNLP](http://wiki.apache.org/tika/TikaAndNER) with [Tika-Python](http://github.com/chrismattmann/tika-python)?

**Comment by elaksana** [comment]
No, we're using a python wrapper over Stanford's CoreNLP:  https://github.com/dasmith/stanford-corenlp-python.

**Comment by chrismattmann** [comment]
darn, that's too bad. would be a great opportunity for coordination.

**Comment by chrismattmann** [comment]
cc @kwhitehall

**Comment by elaksana** [comment]
What do you think, Kim?

**Comment by chrismattmann** [comment]
please note too that the above reference Python library is GPLv2, rather than a permissive license as Tika.

**Comment by kwhitehall** [comment]
Yip, kool feedback @chrismattmann. @elaksana, let's take a look at the PR, then we can go from there.
